edisgo/network/topology.py

```diff
@@ -2811,5 +2811,35 @@ def check_integrity(self):
                 f"are in a different reference system."
             )
 
+    def aggregate_lv_grid_buses_on_station(self, lv_grid_id: int | str) -> None:
```

Comment:
I find it dangerous to do the changes directly in Topology, because the original topology is then lost. I think it would be better to only change the pypsa object.

Reply:
This is the thing used only in ding0, as the last method and should not be used for cost calculations. In ding0 it is important to reduce the topology.